features/mock_for_button_id.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Mock for button id
+  As a user
+  I want to open page with two buttons
+  In order to check payment process for two buttons
+
+  Background:
+    Given JavaScript configuration is set for scenario based on scenario's @config tag
+    And User opens prepared payment form page WITH_ADDITIONAL_BUTTON
+
+  @base_config
+  Scenario: Click on button configured as button id
+    When User fills payment form with defined card VISA_CARD
+    And THREEDQUERY mock response is set to "ENROLLED_Y"
+    And User clicks Pay button - AUTH response is set to "OK"
+    Then THREEDQUERY request was sent only once with correct data
+
+  @base_config
+  Scenario: Click on button configured as additional button
+    When User fills payment form with defined card VISA_CARD
+    And User clicks Additional button
+    Then AUTH and THREEDQUERY requests were not sent
```

Comment:
You should add steps for mock AUTH and THREEDQUERY. Without mock data this step 'Then AUTH and THREEDQUERY requests were not sent' will not work correctly

Reply:
done